packages/models-library/src/models_library/api_schemas_directorv2/clusters.py

```diff
@@ -166,17 +176,19 @@ class Config(BaseCluster.Config):
 
 
 class ClusterPatch(BaseCluster):
-    name: str | None  # type: ignore[assignment]
-    description: str | None
-    type: ClusterTypeInModel | None  # type: ignore[assignment]
-    owner: GroupID | None  # type: ignore[assignment]
-    thumbnail: HttpUrl | None
-    endpoint: AnyUrl | None  # type: ignore[assignment]
-    authentication: ExternalClusterAuthentication | None  # type: ignore[assignment]
+    name: str | None = None  # type: ignore[assignment]
+    description: str | None = None
+    type: ClusterTypeInModel | None = None  # type: ignore[assignment]
+    owner: GroupID | None = None  # type: ignore[assignment]
+    thumbnail: HttpUrl | None = None
+    endpoint: AnyUrl | None = None  # type: ignore[assignment]
+    authentication: ExternalClusterAuthentication | None = None  # type: ignore[assignment]
     access_rights: dict[GroupID, ClusterAccessRights] | None = Field(  # type: ignore[assignment]
-        alias="accessRights"
+        None, alias="accessRights"
     )
```

Comment:
all these default assignments are wrong, they should not be set to default to None or you change the logic.

Reply:
See: https://github.com/ITISFoundation/osparc-simcore/pull/6333#discussion_r1751389736

Reply:
https://docs.pydantic.dev/latest/migration/#unions

Reply:
@sanderegg please, have a look at the documentation of previous Pydantic v1:
https://docs.pydantic.dev/1.10/usage/models/#required-optional-fields